src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -1283,7 +1283,34 @@ void PolicyHandler::OnSnapshotCreated(const BinaryMessage& pt_string) {
     LOG4CXX_ERROR(logger_, "Service URLs are empty! NOT sending PT to mobile!");
     return;
   }
-  SendMessageToSDK(pt_string, urls.front().url.front());
+
+  static size_t current_app = 0;
+  static size_t current_url = 0;
+  if (current_url >= urls.at(current_app).url.size()) {
```

Comment:
@tkireva please change implementation of this code, it is rely confusing. If you have no Ideas how to implement it in other way (without static variables at least), please contact me or Orhan or Ivo or Galiuzov or Vladislav. Feel free to discuss you technical solutions with other developers. Probably create additional task to re factor this  piece of code.